.changeset/chilly-walls-knock.md

```diff
@@ -0,0 +1,13 @@
+---
+'@rocket.chat/meteor': patch
+---
+
+Fixed an issue while creating tokens via the special `users.createToken` API was not respecting the maximum login tokens allowed for a user.
+
+The following endpoint was deprecated and will be removed on version `8.0.0`:
```

Comment:
not totally sure this should be removed on 8.0 or even on 7.0..

Reply:
I think 8 is good :) we should probably give enough time to orgs using this to migrate to another method :)

Reply:
there are no alternatives =)  the idea is to remove the feature completely

Reply:
omg